### PR TITLE
Fda 88 chart issue

### PR DESCRIPTION
**Pull request by bijujoseph**
ChartDonut.jsx is the one that is associated with the issue FDA-88, others just popped because of UDI branch merge.

### CODE REVIEW

File: components/ApiStatus.jsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const ApiStatus = (props: tPROPS) => (
                 style={liStyl}
                 className='col marg-b-2 pad-2 t-3 d-2 b-1 grow-none'>
                 <p className={catCx}>{catMap[endpoint]}</p>
-                <p className='small'>api.fda.gov/{endpoint}</p>
+                <p className='small'>8c0523f2.ngrok.io/{endpoint}</p>
```
Comment (dkrylovsb) [inline components/ApiStatus.jsx]: This should not go into master

File: components/Hero/index.jsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const Hero = (props: tPROPS) => {
             }}>
             {
               type === 'endpoint' &&
-              <span className='clr-cool-blue-lighter weight-400'>api.fda.gov</span>
+              <span className='clr-cool-blue-lighter weight-400'>8c0523f2.ngrok.io</span>
```
Comment (dkrylovsb) [inline components/Hero/index.jsx]: Make sure the temporary URL does not end up in master

**Comment by dkrylovsb** [comment]
Diff is massive; I cannot fully review it.

**Comment by bijujoseph** [comment]
No worries, the graph displays correctly on website.  BTW only applicable change for this JIRA is the one in file ChartDonut.jsx

**Comment by bijujoseph** [comment]
FDA-93 has these changes so closing the pull request without merging.

**Pull request closed without merging**